src/Sylius/Bundle/UiBundle/Resources/views/DataCollector/templateBlock.html.twig

```diff
@@ -66,19 +66,27 @@
 
             {% for event in collector.renderedEvents %}
                 <tr>
-                    <td>{{ event.name }}</td>
+                    <td>
+                        {% for name in event.names %}
+                            {% if loop.first %}<strong>{% endif %}
+                            {{ name }}{% if not loop.last %}, {% endif %}
+                            {% if loop.first %}</strong>{% endif %}
+                        {% endfor %}
+                    </td>
                     <td>{{ '%.0f'|format(event.time * 1000) }}ms</td>
                     <td>
                         {% if event.blocks|length > 0 %}
                             <table>
                                 <tr>
+                                    {% if event.names|length > 1 %}<th>Origin event</th>{% endif %}
```

Comment:
Origin event is only needed if there are multiple events rendered at the same time.